src/prodbg/main/src/windows.rs

```diff
@@ -150,6 +175,7 @@ impl Windows {
                   sessions: &mut Sessions,
                   view_plugins: &mut ViewPlugins,
                   backend_plugins: &mut BackendPlugins) {
+        sleep(Duration::from_millis(100));
```

Comment:
This code should not be here. It is only used to make debugging easier for
me. Main.rs should limit the framerate, not the windows.

2016-07-04 17:48 GMT+08:00 Daniel Collin <notifications@github.com>:

> In src/prodbg/main/src/windows.rs
> <https://github.com/emoon/ProDBG/pull/218#discussion_r69433896>:
>
> > @@ -150,6 +175,7 @@ impl Windows {
> >                    sessions: &mut Sessions,
> >                    view_plugins: &mut ViewPlugins,
> >                    backend_plugins: &mut BackendPlugins) {
> > +        sleep(Duration::from_millis(100));
>
> But sleeping for 100 ms lowers the framerate to 10 fps. Sleeping for 1 ms
> is likely more reasonable.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/emoon/ProDBG/pull/218/files/d4db4aed4c9f97d6991801a031c077eb57d239f0#r69433896>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AANveBffjvqribd6rVCW0Xf7CBpMiqDrks5qSNbYgaJpZM4JELvA>
> .
>